disperser/batcher/metrics.go

```diff
@@ -149,10 +154,23 @@ func NewMetrics(httpPort string, logger logging.Logger) *Metrics {
 		),
 	}
 
+	dispatcherMatrics := DispatcherMetrics{
+		Latency: promauto.With(reg).NewSummaryVec(
+			prometheus.SummaryOpts{
+				Namespace:  namespace,
+				Name:       "attestation_latency_s",
```

Comment:
Could we do milliseconds? It will convert on the dashboard anyway.

Reply:
done